scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -25,7 +26,7 @@ def main():
 
     logger.warn('Date_modified field will be added to all nodes.')
 
-    for node in models.Node.find():
+    for node in models.Node.find(Q('date_modified', 'eq', None)):
```

Comment:
It should query for nodes that don't have the `date_modified` at all.

```
{_id: '123abc'}

// vs. 

{_id: '123abc', date_modified: '2015-.....'}
```

But it sounds like it might be unnecessary?